modules/dnn/src/layers/recurrent_layers.cpp

```diff
@@ -127,6 +127,10 @@ class LSTMLayerImpl CV_FINAL : public LSTMLayer
     bool useAVX2;
 #endif
 
+    // CUDA needs input blobs to be rearranged in a specific way, but some transformations
+    // in ONNXImporter are destructive, so we keep a copy.
+    std::vector<Mat> cudaWorkaround;
```

Comment:
PR is targeted to 3.4 branch, so there is no CUDA.
Is there related PR targeted to 4.x?

Reply:
Yes, there [is](https://github.com/opencv/opencv/pull/20938). In 4.x branch the contents of this vector will be used, but here its purpose is to tell if we need any onnx-related transformations after forward.

Reply:
To avoid confusion we need to rename this field (there is no CUDA code in DNN module for 3.4 branch).
What is about `originalBlobs` or other more clear name?

Reply:
Sure, fixed.